# HG changeset patch
# User Wormnest
# Date 1376508596 -7200
#      Wed Aug 14 21:29:56 2013 +0200
# Node ID 559aa17d516181774b9ce140db453962abce0589
# Parent  61839fdd7a62ec238334ba7003f4e2bf4dce0af2
Add: Introduce news filtering widgets.

diff -r 61839fdd7a62 -r 559aa17d5161 src/widgets/news_widget.h
--- a/src/widgets/news_widget.h	Wed Aug 14 14:46:49 2013 +0200
+++ b/src/widgets/news_widget.h	Wed Aug 14 21:29:56 2013 +0200
@@ -40,6 +40,22 @@
 	WID_MH_STICKYBOX,  ///< Stickybox.
 	WID_MH_BACKGROUND, ///< Background of the window.
 	WID_MH_SCROLLBAR,  ///< Scrollbar for the list.
+
+	WID_MH_NEWS_FILTER_DROPDOWN, ///< News filter dropdown to select the type of news to be shown.
+	/* 
+	 * Vehicletypes taken from station_widget.h but order of ship and airplane was changed.
+	 * They need to be in order of VehicleType due to bit magic.
+	 */
+	WID_MH_TRAIN,          ///< 'TRAIN' button - filter to select trains in vehicle news items.
+	WID_MH_TRUCK,          ///< 'TRUCK' button - filter to select trucks in vehicle news items.
+	WID_MH_BUS,            ///< 'BUS' button - filter to select buses in vehicle news items.
+	WID_MH_SHIP,           ///< 'SHIP' button - filter to select ships in vehicle news items.
+	WID_MH_AIRPLANE,       ///< 'AIRPLANE' button - filter to select aircraft in vehicle news items.
+	WID_MH_VEHICLEALL,     ///< 'ALL' button - select all vehicle news items.
+
+	WID_MH_CARGOALL,       ///< 'ALL' button - select all types of cargo in news items.
+	/* WID_MH_CARGOSTART should be the last widget declared here since there will be multiple cargo widgets. */
+	WID_MH_CARGOSTART,     ///< Widget numbers used for list of cargo types.
 };
 
 #endif /* WIDGETS_NEWS_WIDGET_H */
